packages/app/src/components/nominationInfo/healthProviderDetail.js

```diff
@@ -20,10 +20,12 @@ function HealthProviderDetail(props) {
             <div key={index} className={obj.label === '' ? styles.mobileHide : ''}>
               <label className={styles.label}>{obj.label}</label>
 
-              <Link to={`/nomination/${id}`}>
-                <span className={styles.value} onClick={() => openWindow(obj.value)} key={index}>
-                  {obj.value}
-                </span>
+              <Link to={`/nomination/${id}`} className="green">
+                <u>
```

Comment:
Why is the u tag being used here? This is a pretty esoteric tag that shouldn't be used in most cases: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

Reply:
I updated the PR to include both of your comments. Thanks!